node/runtime/src/lib.rs

```diff
@@ -212,6 +212,7 @@ impl Convert<u128, u128> for CurrencyToVoteHandler {
 
 impl staking::Trait for Runtime {
 	type Currency = Balances;
+	type Time = Timestamp;
```

Comment:
`Moment` is generally better than "Time" and is what was used in `timestamp` module.